groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
+def create_group_category():
+    logging.debug('create_group_category() Called')
+
+    url = canvas_url + '/api/v1/courses/' + course_id + '/group_categories?name=' + urllib.quote_plus(
+            group_category_name)
+    response = api_handler(url, 'Post')
+    if response is None:
+        logging.error('Api call for \'creating group categories\' is not successful')
+        sys.exit(1)
```

Comment:
There doesn't seem to be an exception caught here.

@dovek is right, this doesn't feel like the right place to exit.  I see that the method returns a value in some cases, but the code that calls it never uses the value.  If this method's return value will be used, then it should throw an exception when there's a problem here.